content/docs/EagleEye/installation.mdx

```diff
@@ -24,9 +24,21 @@ Before starting the installation, ensure you have:
 
 ### Phase 1: Deploy from Azure Marketplace
 
+#### Step 0 - Get Tenant ID
```

Comment:
Change this to step 1. I'm not really sure why we're 0 indexing the documentation.

You'll have to update all the following items too.